db/seeds.rb

```diff
@@ -39,4 +32,17 @@
         display_name: "novohispano", role: 0}, { username: "Josh Cheek", email_address: "josh@jumpstartlab.com", password:
         "password", display_name: "josh", role: 1}])
 
-user_content.each { |content| User.create(content) }
+users = user_content.map { |content| User.create(content) }
+
+
+users.first.orders.create(status:0, cart: {"1"=>3, "4"=>1, "9"=>2})
```

Comment:
Space missing after colon.<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.